content/web-api-rest/exercises/_index.md

```diff
@@ -0,0 +1,215 @@
+
+```csharp {linenos=table}
+public class CodingEvent {
+   public int Id { get; set; }
+   public string Title { get; set; }
+   public string Description { get; set; }
+   public DateTime Date { get; set; }
+}
+```
+
+An example of a real `CodingEvent` JSON response to a `GET` request would look like this:
+
+```bash {linenos=table}
+{
+   "id": 1,
+   "title": "Consuming the Coding Events API With Postman",
+   "description": "Learn how to use Postman to interact with the Coding Events API!",
+   "date": "2020-07-24"
+}
+```
+
+Notice how this JSON is just a representation of an instance of the `CodingEvent` model class. The data shape has been converted from a C# object representation to a JSON string representation so it can be transported over HTTP. Recall that we perform this 
+conversion, or serialization, so that our API can output data in a portable format that is language-agnostic.
+
+### The Endpoints
+
+This branch of the API has four endpoints: 
+
+<!-- TODO: These links will need to be updated -->
+1. [GET Coding Events](http://localhost:1313/web-api-rest/exercises/#get-coding-events)
+1. [GET a Single Coding Event](http://localhost:1313/web-api-rest/exercises/#get-a-single-coding-event)
+1. [Create a Coding Event](http://localhost:1313/web-api-rest/exercises/#create-a-coding-event)
+1. [Delete a Coding Event](http://localhost:1313/web-api-rest/exercises/#delete-a-coding-event)
+
+#### Get Coding Events
+
+We'll ask you to consider some details of how to describe these endpoints.
+
+Remember, an endpoint is made up of a path (to the resource) and a method (action to take on the 
+resource). Because we only have one resource, each of our endpoints share a common entry-point path of `/api/events`. Request and response bodies are all in JSON, or more specifically, they have a `Content-Type` header value of `application/json`.
+
+Making a `GET` request to the entry-point of a resource should return a representation of the state of the collection. In our case, this representation is a JSON array with `CodingEvent` elements:
+
+```bash {linenos=table}
+[
+   CodingEvent { ... },
+   ...
+]
+```
+
+If the current state of the collection is empty, then we will just get back an empty JSON array:
+
+```bash {linenos=table}
+   []
+```
+
+{{% notice green "Question" "rocket" %}}
+Using our endpoint shorthand, how would we describe this action?
+
+Some items to consider:
+
+1. What is the HTTP request type being used?
+1. What is the resource path being requested?
+1. Is there a request body being sent? What is included in it?
+1. If the request is successful, what information can we expect to be included in the response?
+{{% /notice %}}
+
+<!-- TODO: Need proper link here -->
+<!-- :ref:[Check your solutions](web-api-ex-1) -->
```

Comment:
```suggestion
{{% expand "Check Your Solution" %}}

{{% expand %}}
```

Reply:
See exercises in classes chapter for reference of adding drop down Check your Solution and add solution.

Reply:
I do not think we need to add a solution to the exercises on this page after looking into it a bit further. The answers to the questions we ask are explained within the question itself. This is more of a check your understanding similar to a reading walkthrough.

Reply:
Thanks for the explanation.